### PR TITLE
prep Exporter and LiveMetrics for beta release

### PR DESCRIPTION
**Pull request by TimothyMothra**

### CODE REVIEW

**Review by rajkumar-rangaraj — APPROVED** [review]

**Review by vishweshbankwar — APPROVED** [review]

File: sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/README.md [hunk 1]
```diff
@@ -1,34 +1,68 @@
...
+Sdk.CreateTracerProviderBuilder()
+    .AddLiveMetrics(o => o.ConnectionString = "InstrumentationKey=00000000-0000-0000-0000-000000000000")
+    .Build();
+```
+
+For a complete example see [`Program.cs`](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/tests/Azure.Monitor.OpenTelemetry.LiveMetrics.Demo/Program.cs)
 
 ### Authenticate the client
 
+Not available in beta.
```
Comment (TimothyMothra) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/README.md:40]: This implementation is in the library, but I haven't had a chance to test it.

File: sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/README.md [hunk 2]
```diff
@@ -1,34 +1,68 @@
...
+- **Azure Application Insights Connection String:** To send telemetry data to the monitoring service you'll need connection string from Azure Application Insights. If you are not familiar with creating Azure resources, you may wish to follow the step-by-step guide for [Create an Application Insights resource](https://docs.microsoft.com/azure/azure-monitor/app/create-new-resource) and [copy the connection string](https://docs.microsoft.com/azure/azure-monitor/app/sdk-connection-string?tabs=net#find-your-connection-string).
+
 ### Install the package
 
+#### Latest Version: [![Nuget](https://img.shields.io/nuget/vpre/Azure.Monitor.OpenTelemetry.LiveMetrics.svg)](https://www.nuget.org/packages/Azure.Monitor.OpenTelemetry.LiveMetrics/)
```
Comment (rajkumar-rangaraj) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/README.md:14]: Will CI allow it to pass?

**Review by rajkumar-rangaraj — APPROVED** [review]

File: sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/README.md (hunk 2 above)
Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/README.md:14]: I was wondering that too. Nope, it just failed, will remove.

File: sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/README.md [hunk 3]
```diff
@@ -1,34 +1,66 @@
...
 
-### Prerequisites
+#### Nightly builds
+
+Nightly builds are available from this repo's [dev feed](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#nuget-package-dev-feed).
+These are provided without support and are not intended for production workloads.
+
+### Add the Live Metrics processor
 
+The following examples demonstrate how to add the `LiveMetricsExtractionProcessor` to your OpenTelemetry configuration.
```
Comment (vishweshbankwar) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/README.md:26]: `LiveMetricsExtractionProcessor` is an internal detail. Call it Live metrics exporter?

Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/README.md:26]: let me change this to "Live Metrics client", which I used everywhere else in the document

**Pull request merged**